frontend/app/player/MessageDistributor/managers/AssistManager.ts

```
@@ -147,7 +154,7 @@ export default class AssistManager {
       })
       socket.on('messages', messages => {
         //console.log(messages.filter(m => m._id === 41 || m._id === 44))
-        jmr.append(messages) // as RawMessage[]
+        !this.timeTravelJump && jmr.append(messages) // as RawMessage[]
```

let them fall into the queue, they will be processed later

---

they will not

---

since they will get added to the q before we get new messages from the mobs file -> we get errors

---

it's better to overlap them and silence errors.  Overprocessed messages is better than missing messages